packages/formation/sass/base/_b-mixins.scss

```diff
@@ -87,19 +87,13 @@
   }
 }
 
-@mixin focus-gold-light-outline($offset: 2) {
+@mixin focus-gold-vivid-outline($offset: 2) {
```

Comment:
Doing this means that we'll have to make a few changes in `vets-website` (and might make this a "breaking change" technically):

- [`m-schemaform.scss`](https://github.com/department-of-veterans-affairs/vets-website/blob/e917d378eba4096724725372333bae696daf90d9/src/platform/forms/sass/_m-schemaform.scss#L212)
- [`facility-locator.scss`](https://github.com/department-of-veterans-affairs/vets-website/blob/e917d378eba4096724725372333bae696daf90d9/src/applications/facility-locator/sass/facility-locator.scss#L62)
- [`edu-benefits.scss`](https://github.com/department-of-veterans-affairs/vets-website/blob/e917d378eba4096724725372333bae696daf90d9/src/applications/edu-benefits/sass/edu-benefits.scss#L46)
- [`10203_stem.scss`](https://github.com/department-of-veterans-affairs/vets-website/blob/e917d378eba4096724725372333bae696daf90d9/src/applications/edu-benefits/10203/sass/10203_stem.scss#L63)

I did it since `$focus-outline` is changing and I thought it would be good for the mixin name to not use the old color.